Symfony/CS/DocBlock/Annotation.php

```diff
@@ -37,7 +37,7 @@ class Annotation
     /**
      * Create a new line instance.
      *
-     * @param Line[] $lines
+     * @param array $lines
```

Comment:
Oh, PHPStorm didn't find the class Line there which is odd since the classes are on the same namespace.